jbpm-bpmn2/src/main/java/org/jbpm/bpmn2/xml/IntermediateThrowEventHandler.java

```diff
@@ -187,6 +187,9 @@ public void handleSignalNode(final Node node, final Element element,
 			final String uri, final String localName,
 			final ExtensibleXmlParser parser) throws SAXException {
 		ActionNode actionNode = (ActionNode) node;
+		String executeActionAfterCompleteMetadata = (String) node.getMetaData().get("executeActionAfterComplete");
+		Boolean executeActionAfterComplete = executeActionAfterCompleteMetadata != null ? Boolean.parseBoolean(executeActionAfterCompleteMetadata) : false;
```

Comment:
It could be simplified without a ternary operator:
```suggestion
		Boolean executeActionAfterComplete = executeActionAfterCompleteMetadata != null && Boolean.parseBoolean(executeActionAfterCompleteMetadata);
```